call-home/assets/ca_cert/Caringo_Root_CA_2015.pem

```diff
@@ -0,0 +1,35 @@
+-----BEGIN CERTIFICATE-----

```

Comment:
This certificate is not required and _must_ be removed.

Reply:
Done.